FitNesseRoot/properties

```diff
@@ -1,9 +1,10 @@
-#FitNesse properties
-#Sun Mar 18 20:20:29 EDT 2007
-WikiImportFormatUpdate=applied
-PropertiesToXmlUpdate=applied
-FilesAttributeUpdate=applied
+##FitNesse properties
```

Comment:
Looks like it's format only changes here, we better exclude it.

Reply:
Yea, I would guess intellij reformatted those. I have pushed an update to change that back to the master branch format.